server/configuration/base.yaml

```diff
@@ -2,7 +2,11 @@ server:
   port: 8000
   # debug_no_access_control: true # enable this if you want to ignore API authorisation (dummy user will be used for all operations)
   # debug_cors_permissive: true # enable this to allow any origin for development purposes
-  cors_origins: [http://localhost:3003, https://demo-open.msupply.org, http://localhost:8000] # Used to set the allowed Origin in Cross Origin Requst Security
+  cors_origins: [
+      http://localhost:3003,
+      https://demo-open.msupply.org,
+      http://localhost:8000,
+    ] # Used to set the allowed Origin in Cross Origin Requst Security
 sync:
```

Comment:
I'm using the sync config from the database.. so I'll still have to update `base.yaml` on every checkout in order to comment out the sync settings. I know that I said I agreed with Clemens and that we should still have base.yaml etc.. but I'm realising the implications now. 

Retaining base.yaml in the repo will still be irritating for me..

Reply:
Do we need sync settings in base.yaml? We could just remove them and people can add them if required in local.yaml?

Reply:
Removed sync from base.yaml